torchdynamo/symbolic_convert.py

```diff
@@ -865,6 +882,20 @@ def LOAD_ATTR(self, inst):
                 self.push(UserFunctionVariable(member, **options))
             else:
                 unimplemented("PythonModuleVariable attribute")
+        elif isinstance(obj, UserClassInstanceVariable):
+            attr = getattr(obj.value, name)
```

Comment:
1) `getattr()` can execute arbitrary code via properties.  So we need to check for that with inspect.getattr_static().   
2) If the attribute is on the class (and we guard based on the type) then we can know it at compile time.  This is useful for things like method calls.  The other cases above mostly fall into this case.
3) If the attribute is on the instance, then we need to read it at runtime. This code is incorrectly reading the compile-time value.

For case 3, we will need to add guards to handle this correctly.  For `foo.bar = True`, we have a guard checking the type of `foo`, but just because we know the type of `foo` does not mean we know the type of `foo.bar`.   This is solvable, but requires careful accounting of guards.